package.json

```diff
@@ -29,19 +29,20 @@
     "packages/*"
   ],
   "scripts": {
-    "check:types": "npm run clean && tsc --build ./tsconfig.json && tsc -p tsconfig.unit.json --noEmit --module preserve --moduleResolution bundler && tsc -p e2e/tsconfig.json --noEmit --skipLibCheck",
-    "clean": "rm -rf dist out packages/*/tsconfig.tsbuildinfo",
-    "compile": "npm run check:types && node scripts/esbuild.js",
-    "compile:prod": "npm run check:types && npm run test:lint && node scripts/esbuild.js --production",
+    "build:ts": "npm run clean && tsc --build ./tsconfig.json && tsc -p tsconfig.unit.json --noEmit --module preserve --moduleResolution bundler && tsc -p e2e/tsconfig.json --noEmit --skipLibCheck",
+    "clean": "rm -rf dist out e2e-testing/out tsconfig.tsbuildinfo e2e-testing/tsconfig.tsbuildinfo packages/*/tsconfig.tsbuildinfo",
+    "compile": "npm run build:ts && node scripts/esbuild.js",
+    "compile:prod": "npm run build:ts && npm run test:lint && node scripts/esbuild.js --production",
     "icon:gen": "node icons/generate.mjs",
     "package:latest": "vsce package -o releases/vscode-deephaven-latest.vsix",
     "publish:prerelease": "./scripts/publish-prerelease.sh",
     "report:ctrfmerge": "ctrf merge e2e/reports --keep-reports --output-dir test-reports --output ctrf-report.json",
-    "report:ctrfsummary": "github-actions-ctrf test-reports/ctrf-report.json",
+    "report:ctrfsummary": "github-actions-ctrf summary test-reports/ctrf-report.json && github-actions-ctrf failed test-reports/ctrf-report.json",
     "report:junit2ctrf": "junit-to-ctrf test-reports/vitest.junit.xml --output test-reports/ctrf-report.json",
     "report:prcomment": "mkdir -p ./pr-comment && cp $GITHUB_STEP_SUMMARY pr-comment/pr-comment.html",
     "test:ci": "npm run compile && npm run test:lint && npm run test:unit",
-    "test:e2e": "npm run compile && cd e2e && wdio run ./wdio.conf.ts",
+    "test:e2e": "npm run build:ts && ./scripts/e2e.sh",
```

Comment:
Instead of using `&&` use [`concurrently`](https://www.npmjs.com/package/concurrently). `&&` isn't cross platform compatible.

Reply:
I'd be curious to understand what incompatibilities there may be, as I was under the impression this works on unix like environments and even in Windows CMD.
 
Also worth mentioning that this project already assumes you have to be in a Unix like environment that supports bash scripts, so maybe we are constrained in a way that incompatibilities don't apply?

Reply:
I thought there was incompatibility with Windows cmd.exe, I may be incorrect about that. Had other issues with `&&` though, see a previous change where I ran into issues with environment variables as part of the statements: https://github.com/deephaven-ent/iris/pull/1910

Reply:
I know there are definitely incompatibilities with the single `&` operator between Windows cmd.exe and linux. Could also be some with `&&` that I haven't run into yet. The environment variable issue also makes sense though. 

I'd propose we just leave it alone for now until we run into an issue since Windows users will already need to use WSL. I also don't mind creating a ticket if you think it should be addressed more proactively.